src/lexer.c

```diff
@@ -280,21 +282,21 @@ void tryParseExportStatement () {
       ch = commentWhitespace();
       if (ch == 'f' && str_eq3(pos + 1, 'r', 'o', 'm')) {
         pos += 4;
-        readImportString(commentWhitespace());
+        readImportString((char16_t*)startPos, commentWhitespace());
```

Comment:
Casting here to remove this error I was seeing on compilation: 

```
src/lexer.c:285:26: warning: passing 'const char16_t *' (aka 'const unsigned short *') to parameter
      of type 'char16_t *' (aka 'unsigned short *') discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
        readImportString(startPos, commentWhitespace());
```

If there's a better solution that you'd prefer lmk

Reply:
I think it would be better to make the argument in `readImportString` const.

Reply:
done